src/Fabulous.XamarinForms/Views/Controls/Image.fs

```diff
@@ -9,22 +11,35 @@ type IImage =
 module Image =
     let WidgetKey = Widgets.register<Image> ()
 
-    let Source =
-        Attributes.defineBindable<ImageSource> Image.SourceProperty
-
     let Aspect =
         Attributes.defineBindable<Aspect> Image.AspectProperty
 
+    let IsAnimationPlaying =
+        Attributes.defineBindable<bool> Image.IsAnimationPlayingProperty
+
+    let IsOpaque =
+        Attributes.defineBindable<bool> Image.IsOpaqueProperty
+
+    let Source =
+        Attributes.defineAppThemeBindable<string> Image.SourceProperty
+
 [<AutoOpen>]
 module ImageBuilders =
     type Fabulous.XamarinForms.View with
         static member inline Image<'msg>(imageSource: ImageSource, aspect: Aspect) =
             WidgetBuilder<'msg, IImage>(
                 Image.WidgetKey,
-                Image.Source.WithValue(imageSource),
+                (Attributes.defineBindable<ImageSource> Image.SourceProperty)
```

Comment:
If you call `define` here, it will create a different attribute definition each time you use `Image(...)`.
This will break the diffing process..

Reply:
So you should define only 1 `Source` attribute of type `ImageSource`, and then have 2 builders:
- ImageSource * Aspect
- path: string * Aspect
- uri: Uri * Aspect
- stream: Stream * Aspect

They will all call the 1st builder.

Reply:
I’m temped to suggest that we add another widget to support Themedimage to add support for light and dark themes

Reply:
Removed this and use only 1  source